app/client/src/cytoscape-graphml.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'cytoscape-graphml';
```

Comment:
I don't think you should need this here, as well as declaring in `shims-vue.d.ts`?